src/main/markdown/release-notes.md

```diff
@@ -61,6 +62,39 @@ The GWT Release Notes
 * * *
 
 <a id="Release_Notes_Current"></a>
+## <a id="Release_Notes_2_8_2"></a> Release Notes for 2.8.2
+
+### Highlights
+- Supports running in Java 9.
+
+- Chrome 61 removed functionality that had been used for reading the absolute
+top/left values. The internal implementation has been updated to reflect modern
+standards.
+
+### Bug fixes
+- LookupMethodCreator creates too large method
+- NativeRegExp should use iframe instance, fixing Edge JIT bug
+- JsProperty getter/setter sometimes were reported as incompatible
+- Instantiating native JsTypes from JSNI results in InternalCompilerException
+- Remove the SUBSIZED characteristic from filtered streams
+- Internal compiler exception when using native JsType varargs in a JsMethod
+- Regression in String.toLowerCase and toUpperCase
+- Missing bounds check in String.charAt
+- Fix AIOOBE when compiling method references involving varargs.
+- Apply HtmlUnit workaround ensuring that window.onerror is called correctly
+
+### Miscellanous
+- Migrated lang/jre emulation JSNI to JsInterop to share with J2CL
+- Added ErrorProne to gwt builds
+- Improved compliance with CSP
+- Uncaught Exception Handler now can receive window.onerror
```

Comment:
IIRC, Daniel asked that we mark this as a "potentially breaking" change; how about moving this to the highlights section? (and it's not a "can" but "will, by default (use XXX to revert to previous behavior)"